File: app/src/main/java/pavel/bogrecov/omertex/utils/NetworkUtils.java
package pavel.bogrecov.omertex.utils;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

import pavel.bogrecov.omertex.presentation.app.AppClass;

public class NetworkUtils {


    public static boolean isNetworkAvailable() {
        ConnectivityManager connectivityManager =
                (ConnectivityManager) AppClass.getContext().getSystemService(Context.CONNECTIVITY_SERVICE);
        NetworkInfo activeNetworkInfo = connectivityManager.getActiveNetworkInfo();
        return activeNetworkInfo != null && activeNetworkInfo.isConnected();
    }

}

File: app/src/main/java/pavel/bogrecov/omertex/utils/NetworkUtils.kt
package pavel.bogrecov.omertex.utils

import android.content.Context
import android.net.ConnectivityManager
import pavel.bogrecov.omertex.presentation.app.AppClass

object NetworkUtils {


    val isNetworkAvailable: Boolean
        get() {
            val connectivityManager = AppClass.getContext().getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
            val activeNetworkInfo = connectivityManager.activeNetworkInfo
            return activeNetworkInfo != null && activeNetworkInfo.isConnected
        }

}
